website/content/docs/connect/config-entries/service-resolver.mdx

```diff
@@ -476,6 +527,19 @@ spec:
           description:
             'Specifies the destination datacenter to resolve the service from.',
         },
+        {
+          name: 'Peer',
+          type: 'string: ""',
+          description:
+              `Specifies the destination cluster peer to resolve the target service name from.
+                  Ensure that [intentions are defined](/docs/connect/cluster-peering/create-manage-peering#authorize-services-for-peers)
+                  on the peered cluster to allow the source service to call the upstream service. When
```

Comment:
@erichaberkorn : does this seem like a helpful clarification (that the upstream we're referring to is the service in this redirect clause)?

```suggestion
                  on the peered cluster to allow the source service to access this redirect target service as an upstream.
```